fastlane/Fastfile

```diff
@@ -1089,9 +1089,9 @@ end
 # the account index, and values are either "free" or BUILDKITE_BUILD_NUMBER, in case if account
 # was occupied by a build. Returns the hash.
 # Example:
-#   get_test_accounts_hash
+#   fetch_test_accounts_hash
 #     => {"0"=>"4079", "1"=>"free", "2"=>"free", "3"=>"free"}
-def get_test_accounts_hash
+def fetch_test_accounts_hash
```

Comment:
I think the RuboCop rule here recommends to omit `get_` for getters, leaving them from `get_resource` to simply `resource`.

However, I just wanted to say I like how you made this `fetch_...`. It makes is clearer that something more is going on under the hood (i.e., an HTTP call)